syft/workers/websocket_client.py

```diff
@@ -34,11 +36,18 @@ def __init__(
         # TODO get angry when we have no connection params
         self.port = port
         self.host = host
-        self.uri = f"ws://{self.host}:{self.port}"
 
         # creates the connection with the server which gets held open until the
         # WebsocketClientWorker is garbage collected.
-        self.ws = websocket.create_connection(self.uri, max_size=None)
+        # Secure flag adds a secure layer applying cryptography and authentication
+        self.uri = f"ws://{self.host}:{self.port}"
+        if secure:
+            self.uri = f"wss://{self.host}:{self.port}"
+            ssl_settings = {"cert_reqs": ssl.CERT_NONE}
```

Comment:
This line should only be used in development environments.
It allows to work with self-signed certificates facilitating the development.

Reply:
FYI - over the last little bit I've been working on a Heroku based flask app worker - when the time comes I'd love to pick your brain on how to integrate these - especially the security pieces.

Reply:
I would be happy to help. :smiley:

Reply:
Excellent!